docs/specifications/overview.md

```diff
@@ -2,10 +2,7 @@
 
 ## Synopsis
 
-- Numaflow allows developers with basic knowledge of Kubernetes but
-  without any special knowledge of data/stream processing to easily
-  create massively parallel data/stream processing jobs using a
-  programming language of their choice.
+- Numaflow allows developers without any special knowledge of data/stream processing to easily create massively parallel data/stream processing jobs using a programming language of their choice, with just basic knowledge of Kubernetes.
```

Comment:
please keep it multiline, else in IDEs the line will be too long. Markdown rendering will merge these multiline together.

Reply:
done!